Fix warning "	warning NU5118: File 'Microsoft.AspNetCore.AzureAppServices.HostingStartup.deps.json' is not added because the package already contains file 'Microsoft.AspNetCore.AzureAppServices.HostingStartup.deps.json'"

Part of https://github.com/dotnet/aspnetcore/issues/22240

Happens during the SiteExtension build. Full warning text below:

> F:\workspace\_work\1\s\.dotnet\sdk\5.0.100-preview.5.20258.4\Sdks\NuGet.Build.Tasks.Pack\build\NuGet.Build.Tasks.Pack.targets(198,5): warning NU5118: File 'F:\workspace\_work\1\s\.packages\Microsoft.AspNetCore.AzureAppServices.SiteExtension.5.0.x64\5.0.0-preview-6-20275-3\content\additionaldeps\shared\Microsoft.AspNetCore.App\5.0.0\Microsoft.AspNetCore.AzureAppServices.HostingStartup.deps.json' is not added because the package already contains file 'content\additionaldeps\shared\Microsoft.AspNetCore.App\5.0.0\Microsoft.AspNetCore.AzureAppServices.HostingStartup.deps.json' [F:\workspace\_work\1\s\src\SiteExtensions\LoggingAggregate\src\Microsoft.AspNetCore.AzureAppServices.SiteExtension\Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj]

> F:\workspace\_work\1\s\.dotnet\sdk\5.0.100-preview.5.20258.4\Sdks\NuGet.Build.Tasks.Pack\build\NuGet.Build.Tasks.Pack.targets(198,5): warning NU5118: File 'F:\workspace\_work\1\s\.packages\Microsoft.AspNetCore.AzureAppServices.SiteExtension.3.1.x86\3.1.3-servicing-20163-14\content\additionaldeps\shared\Microsoft.AspNetCore.App\3.1.0\Microsoft.AspNetCore.AzureAppServices.HostingStartup.deps.json' is not added because the package already contains file 'content\additionaldeps\shared\Microsoft.AspNetCore.App\3.1.0\Microsoft.AspNetCore.AzureAppServices.HostingStartup.deps.json' [F:\workspace\_work\1\s\src\SiteExtensions\LoggingAggregate\src\Microsoft.AspNetCore.AzureAppServices.SiteExtension\Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj]

--- a/src/SiteExtensions/LoggingAggregate/src/Microsoft.AspNetCore.AzureAppServices.SiteExtension/Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj
+++ b/src/SiteExtensions/LoggingAggregate/src/Microsoft.AspNetCore.AzureAppServices.SiteExtension/Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj
@@ -66,6 +66,9 @@
     <ItemGroup>
       <!-- LB.csproj package content is bundled into this one. -->
       <ContentFilesToPack Include="$(NugetPackageRoot)\%(PackageReference.Identity)\%(PackageReference.Version)\content\**\*.*" />
+      <!-- The x64 & x86 SiteExtension packages have identical deps.json files - we only include the x64 files to avoid build warnings -->
+      <ContentFilesToPack Exclude="$(NugetPackageRoot)\Microsoft.AspNetCore.AzureAppServices.SiteExtension.3.1.x86\$(MicrosoftAspNetCoreAzureAppServicesSiteExtension31PackageVersion)\content\**\*.deps.json" />
+      <ContentFilesToPack Exclude="$(NugetPackageRoot)\Microsoft.AspNetCore.AzureAppServices.SiteExtension.5.0.x86\$(PackageVersion)\content\**\*.deps.json" />
 
       <!-- Temporarily skip the common files -->
       <FilteredContentFilesToPack Include="@(ContentFilesToPack)" Condition="'%(RecursiveDir)' != ''" />

--- a/src/SiteExtensions/LoggingAggregate/src/Microsoft.AspNetCore.AzureAppServices.SiteExtension/Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj
+++ b/src/SiteExtensions/LoggingAggregate/src/Microsoft.AspNetCore.AzureAppServices.SiteExtension/Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj
@@ -65,10 +65,10 @@
 
     <ItemGroup>
       <!-- LB.csproj package content is bundled into this one. -->
-      <ContentFilesToPack Include="$(NugetPackageRoot)\%(PackageReference.Identity)\%(PackageReference.Version)\content\**\*.*" />
       <!-- The x64 & x86 SiteExtension packages have identical deps.json files - we only include the x64 files to avoid build warnings -->
-      <ContentFilesToPack Exclude="$(NugetPackageRoot)\Microsoft.AspNetCore.AzureAppServices.SiteExtension.3.1.x86\$(MicrosoftAspNetCoreAzureAppServicesSiteExtension31PackageVersion)\content\**\*.deps.json" />
-      <ContentFilesToPack Exclude="$(NugetPackageRoot)\Microsoft.AspNetCore.AzureAppServices.SiteExtension.5.0.x86\$(PackageVersion)\content\**\*.deps.json" />
+      <ContentFilesToPack 
+          Include="$(NugetPackageRoot)\%(PackageReference.Identity)\%(PackageReference.Version)\content\**\*.*" 
+          Exclude="$(NugetPackageRoot)\Microsoft.AspNetCore.AzureAppServices.SiteExtension.*.x86\**\Microsoft.AspNetCore.AzureAppServices.HostingStartup.deps.json"/>
 
       <!-- Temporarily skip the common files -->
       <FilteredContentFilesToPack Include="@(ContentFilesToPack)" Condition="'%(RecursiveDir)' != ''" />

